DancingGoat/Views/Shared/_Layout.cshtml

```diff
@@ -1,121 +1,195 @@
-﻿@using System.Configuration
```

Comment:
Too much change in this file, minimize the changes ... use partial view if you need to inject specific HTML